test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -1566,17 +1561,9 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 
 		It("[test_id:50966]verify offlined parameter accepts multiple ranges of cpuid's", func() {
 			var reserved, isolated, offlined []string
-			//This map is of the form numaNode[core][cpu-siblings]
-			var numaCoreSiblings map[int]map[int][]int
-			for _, node := range workerRTNodes {
-				numaInfo, err := nodes.GetNumaNodes(&node)
-				Expect(err).ToNot(HaveOccurred())
-				if len(numaInfo) < 2 {
-					Skip(fmt.Sprintf("This test need 2 NUMA nodes.The number of NUMA nodes on node %s < 2", node.Name))
-				}
-			}
-			for _, node := range workerRTNodes {
-				numaCoreSiblings, err = nodes.GetCoreSiblings(&node)
+			numaTopology := copyNumaCoreSiblings(numaCoreSiblings)
+			if len(numaTopology) < 2 {
```

Comment:
this is not aligned as the 4.14 code (comparing https://github.com/openshift/cluster-node-tuning-operator/pull/1035/files)

Reply:
@yanirq addressed the above in latest commit.